src/webgl/p5.RendererGL.js

```diff
@@ -1246,7 +1246,7 @@ p5.RendererGL.prototype._triangulate = function(contours) {
   // since only doing 2d tesselation here, provide z=1 normal to skip
   // iterating over verts only to get the same answer.
   // comment out to test normal-generation code
-  this._tessy.gluTessNormal(0, 0, 1);
```

Comment:
Sorry I think I might have been a bit unclear. I meant to say that commenting out this line will allow us to generate coplanar concave shapes regardless of the plane that they are created on **but** it isn't a good idea because then normals are calculated every frame. Requiring users to make concave filled WEBGL shapes on the z-plane will make this line correct. If we don't tessellate when vertices are not on the same z-plane then this code will only ever be reached when this line is correct. Does that make sense?